File: C/Palindrome Number.c

bool isPalindrome(int x){
    long l,s,r;
    l=x;
    s=0;
    if(l>=0)
while(l!=0)
{
    r=l%10;
    s=s*10+r;
    l=l/10;
}
    if(s==x)
        return true;
    else
        return false;
}

File: Java/Palindrome Number.java
class Solution {
    public boolean isPalindrome(int x) 
    {
        int s=0,r,n;
        n=x;
        while(n>0)
        {
            r=n%10;
            s=s*10+r;
            n=n/10;
        }
        return(x==s);
    }
}